docs/running-headscale-container.md

```diff
@@ -66,6 +66,8 @@ db_type: sqlite3
 db_path: /etc/headscale/db.sqlite
 ```
 
+Change the `unix_socket: /var/run/headscale/headscale.sock` to `unix_socket: ./headscale.sock` for gRPC socket if you encounter `Error starting server error="failed to set up gRPC socket` related error 
```

Comment:
This is currently discussed in #1417

Reply:
Maybe we should introduce it as a temporarily fix for docker users?

Reply:
This is now obsolete due to #1450.